lib/galaxy/tools/parameters/wrapped_json.py

```diff
@@ -62,8 +62,14 @@ def _data_input_to_path(v):
             json_value = _data_input_to_path(value)
         elif handle_files == "skip":
             return SKIP_INPUT
-        else:
-            raise NotImplementedError()
+        elif handle_files == "OBJECT":
+            if value:
+                if isinstance(value, list):
+                    value = value[0]
+                return _hda_to_object(value)
+            else:
+                return None
+        raise NotImplementedError()
```

Comment:
```suggestion
        else:
            raise NotImplementedError()
```
This broke the inputs_as_json_with_paths tool, I think